src/app/controller/pipes/vessel-tooltip.pipe.ts

```diff
@@ -8,15 +7,20 @@ import { Vessel } from "../../model/portCall/vessel";
 })
 export class VesselTooltipPipe implements PipeTransform {
 
-  transform(vessel: Vessel): string {
+  transform(vessel: Vessel, milesRemainingToDestination: number): string {
+
+    let tooltip = "";
     if (vessel) {
-      var tooltip = "";
       if (vessel.type) tooltip += `Type: ${vessel.type}\n`;
       if (vessel.width) tooltip += `Width: ${vessel.width}\n`;
       if (vessel.length) tooltip += `Length: ${vessel.length}\n`;
       if (vessel.vesselCallSignNumber) tooltip += `Call sign: ${vessel.vesselCallSignNumber}\n`;
-      return tooltip
+      if (milesRemainingToDestination) tooltip += `milesRemainingToDestination: ${milesRemainingToDestination}\n`;
+      if (vessel.vesselDraft) tooltip += `Draft: ${vessel.length}\n`;
```

Comment:
I would assume we need a unit of measuer here (Feet or Meters) - the value can be read in the `vesselDraftUnit` attribute (might not be implemented yet in backend)

Reply:
same as above